src/libMessage/Messenger.cpp

```diff
@@ -3233,6 +3493,42 @@ bool Messenger::GetLookupGetStartPoWFromSeed(const vector<unsigned char>& src,
   return true;
 }
 
+bool Messenger::SetLookupSetStartPoWFromSeed(
```

Comment:
I think for empty messages we can still sign something, because it's still important to verify the lookup in these cases. We can either just define some dummy bytes to sign, or have something more meaningful like maybe the epoch number.

Reply:
I think can do.